pkg/corerp/frontend/controller/applications/testdata/graph-app-httproute-in.json

```diff
@@ -0,0 +1,55 @@
+[
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/httpRoutes/sql-rte",
+        "name": "sql-rte",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "provisioningState": "Succeeded"
+        },
+        "type": "Applications.Core/httpRoutes"
+    },
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-app-ctnr",
+        "name": "sql-app-ctnr",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "connections": {
+                "sql": {
+                    "source": "/planes/radius/local/resourcegroups/default/providers/Applications.Datastores/sqlDatabases/sql-db"
+                }
+            },
+            "provisioningState": "Succeeded",
+            "status": {
+                "outputResources": {
+                    "id": "/some/thing/else",
+                    "localId": "something"
+                }
+            }
+        },
+        "type": "Applications.Core/containers"
+    },
+    {
+        "id": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/containers/sql-ctnr",
+        "name": "sql-ctnr",
+        "properties": {
+            "application": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/Applications/myapp",
+            "container": {
+                "ports": {
+                    "web": {
+                        "port": 8080,
+                        "protocol": "TCP",
+                        "provides": "/planes/radius/local/resourcegroups/default/providers/Applications.Core/httpRoutes/sql-rte"
+                    }
+                }
+            },
+            "provisioningState": "Succeeded",
+            "status": {
+                "outputResources": {
+                    "id": "/some/thing/else",
+                    "localId": "something"
+                }
+            }
+        },
+        "type": "Applications.Core/containers"
+    }
+]
```

Comment:
Just one question: should the files have an extra line at the end? For formatting purposes.